realgud/common/track.el

```diff
@@ -740,6 +760,10 @@ find a location. non-nil if we can find a location.
 	  nil))
       ))
     )
+(defun realgud-set-eval-string-to-buffer-local (command-hash)
+  "Sets the eval string as a buffer local variable from the COMMAND-HASH."
+  (set (make-local-variable 'realgud-eval-string)
+       (replace-regexp-in-string "%s" "" (or (gethash "eval" command-hash) "No Eval String"))))
 
```

Comment:
whoops, your comment got lost cause i did a push -f to fix a bug, here is what @rocky said

This feels familiar. In the various debuggers I've written I often have a variable "last command" variable that I set. This is whether it was an "eval" or something else.

So if I understand this correctly, one might do that here and generalize this use. And it might be useful in addressing another long-standing problem and feature request that people have:

Many debuggers like pdb don't acknowledge success of setting a breakpoint. Only failure. Because of this, numerous people have suggested that realgud look for the absense of a complaint after setting a breakpoint in order to store information that a new breapoint has been added so that it can update its internal state.

If I have this correct this same kind of thing might be useful there. Right?

Reply:
yes, right now my processing of the output in the comint-hook is the first time thats been done right?  So if we process that output then we can start doing things like saying "oh, the last command was a breakpoint, and the output was empty so the breakpoint was a success"

Reply:
Yes I think it is. 

When the dust settles, I'll pull all of this down locally and look at it as not a set of diffs, so I'll have a better sense then.